lib/carbon/conf.py

```diff
@@ -246,7 +246,7 @@ def postOptions(self):
             print "Error: missing required config %s" % storage_schemas
             sys.exit(1)
 
-        if settings.CACHE_WRITE_STRATEGY not in ('sorted', 'max', 'naive'):
+        if settings.CACHE_WRITE_STRATEGY not in ('timesorted', 'sorted', 'max', 'naive'):
```

Comment:
Is there a reason `random` isn't included here?  It seems like this could be avoided (and code in `cache.MetricCache()` simplified to just `write_strategy = write_strategies.get(settings.CACHE_WRITE_STRATEGY)`) if we defined `cache.write_strategies` as a `dict` mapping the string values to classes and instead checked if `settings.CACHE_WRITE_STRATEGY` was a key in that `dict`.